modules/contact/src/AugmentedLagrangianContactProblem.C

```diff
@@ -68,6 +69,7 @@ AugmentedLagrangianContactProblem::checkNonlinearConvergence(std::string & msg,
                                                              const Real /*div_threshold*/)
 {
 
+  Real my_dtol = std::numeric_limits<int>::max();
```

Comment:
Why this `dtol` takes a different way? It should use `dtol` passed from `PetscSupport.C`

Reply:
Could you please explain a little bit more?

Reply:
I am not very familiar with the contact module and did not want to disrupt any workflows.  I can change it to use `PETScSupport.C` if you like.  Likewise,`ReferenceResidualProblem.C` will need to be updated.

Reply:
@fdkong, do you want me to add a function to `ReferenceResidualProblem.C` to check the divergence of each variable similar to `checkConvergenceIndividVars?`

Reply:
@fdkong, this is a follow-up regarding my previous question?  In `ReferenceResidualProblem.C`, each individual variable is checked for convergence.  Do you want the divergence to be checked for each individual variable as well using a new function called `checkDivergenceIndividVars?`